README.md

```diff
@@ -110,22 +111,26 @@ Clone this repository to your desired folder:
 
 Install this project with:
 
-```sh
-- In the first commit of your feature branch create a .github/workflows folder and add a copy of [.github/workflows/linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) to that folder.
+
+- In the first commit of your feature branch create a .github/workflows folder and add a copy of [.github/workflows/linters.yml](https://github.com/microverseinc/linters-config/blob/master/react-redux/.github/workflows/linters.yml) to that folder.
 - create a .gitignore file and add 'node_modules' to it
-- run 'npm init -y'
-- run 'npm install --save-dev hint@7.x'
-- Copy [hintrc](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.hintrc) to the root directory of your project.
-- run 'npx hint .'
-- Fix validation errors.
-- run 'npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x'
-- Copy [stylelintrc.json](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.stylelintrc.json) to the root directory of your project.
-- Run 'npx stylelint "\*_/_.{css,scss}"'
-- fix linter errors
-- run "npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x"
-- Copy [eslintrc.json](https://github.com/microverseinc/linters-config/tree/master/html-css-js)
-- Run npx eslint . on the root of your directory of your project
-- Fix linter error.
+- run 'npm init -y
+
+- Run
+```sh
+npm install --save-dev eslint@7.x eslint-config-airbnb@18.x eslint-plugin-import@2.x eslint-plugin-jsx-a11y@6.x eslint-plugin-react@7.x eslint-plugin-react-hooks@4.x @babel/eslint-parser@7.x @babel/core@7.x  @babel/plugin-syntax-jsx@7.x  @babel/preset-react@7.x @babel/preset-react@7.x
+```
+- Copy [.eslintrc.json](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json) and [.babelrc](https://github.com/microverseinc/linters-config/blob/master/react-redux/.babelrc) to the root directory of your project.
+- Run `npx eslint "**/*.{js,jsx}"` on the root of your directory of your project.
+- Fix linter errors.
+- Run
+```sh
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+```
+- Copy [.stylelintrc.json](https://github.com/microverseinc/linters-config/blob/master/react-redux/.stylelintrc.json) to the root directory of your project.
+- Run `npx stylelint "**/*.{css,scss}"` on the root of your directory of your project.
+- Fix linter errors.
+
```

Comment:
I think when the user runs npm install on the terminal, they get every required package installed and ready for use. Kindly simplify this section. 👍🏾